hls4ml/converters/keras/qkeras_layers.py

```diff
@@ -28,26 +28,26 @@ def parse_qdense_layer(keras_layer, input_names, input_shapes, data_reader, conf
 @keras_handler('QConv1D', 'QConv2D')
 def parse_qconv_layer(keras_layer, input_names, input_shapes, data_reader, config):
     assert('QConv' in keras_layer['class_name'])
-    
-    if int(keras_layer['class_name'][-2]) == 1:
+
+    if "1D" in keras_layer['class_name']:
         layer, output_shape = parse_conv1d_layer(keras_layer, input_names, input_shapes, data_reader, config)
-    elif int(keras_layer['class_name'][-2]) == 2:
+    elif "2D" in keras_layer['class_name']:
```

Comment:
These changes are not specific to the QConv2DBatchnorm - please remove them, but consider making a new PR with these